### PR TITLE
systemd-ask-password: add page

### PR DESCRIPTION
**Pull request by leenate**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- Version - systemd 247 (247.3-7)

### CODE REVIEW

File: pages/linux/systemd-ask-password.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Use a kernel keyring key name as a cache for the password:
+
+`systemd-ask-password --keyname={{key_name}} "{{message}}"`
+
+- Set a custom timeout for the password query:
+
+`systemd-ask-password --timeout={{seconds}} "{{message}}"`
+
+- Force the use of agent system and never ask on current TTY:
```
Comment (kbdharun) [inline pages/linux/systemd-ask-password.md:22]:
```suggestion
- Force the use of an agent system and never ask on current TTY:
```

File: pages/linux/systemd-ask-password.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`systemd-ask-password --no-tty "{{message}}"`
+
+- Store a password in kernel keyring without displaying it:
```
Comment (kbdharun) [inline pages/linux/systemd-ask-password.md:26]:
```suggestion
- Store a password in the kernel keyring without displaying it:
```

**Review by kbdharun — APPROVED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

**Comment by leenate** [comment]
> Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

Thanks for the suggestions; I think those were improvements!

**Review by gutjuri — APPROVED** [review]

**Pull request merged**